.vscode/settings.json

```diff
@@ -17,14 +28,20 @@
   "[markdown]": {
     "editor.formatOnSave": false
   },
+  "css.lint.unknownAtRules": "ignore",
   "prettier.enable": true,
   "prettier.prettierPath": "",
   "prettier.configPath": "./frontend/.prettierrc",
   "files.eol": "\n",
   "[python]": {
     "editor.defaultFormatter": "ms-python.black-formatter"
   },
-  "pylint.args": ["--load-plugins=pylint_django", "--rcfile", "${workspaceRoot}/.pylintrc"],
-  "black-formatter.args": ["--line-length=119"],
-  "black-formatter.path": ["${workspaceRoot}/venv/bin/black"]
-}
+  "pylint.args": [
+    "--load-plugins=pylint_django",
+    "--rcfile",
+    "${workspaceFolder}/.pylintrc"
+  ],
+  "black-formatter.args": [
+    "--line-length=119"
+  ],
+}
```

Comment:
On ajoute une dernière ligne vide à la fin?

Reply:
Je me demande si c'est pas pre-commit qui a ajouté la ligne tout seul pour valider le JSON ?

Reply:
Là je crois qu'il n'y en a pas de dernière ligne (d'où la petite icône :no_entry_sign: à la fin du fichier sur Github). Ça serait pas mal de l'ajouter (en générale [recommandé](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file) même si ce n'est pas la fin du monde de ne pas l'avoir)

Reply:
Ah oui my bad ! D'accord avec cette bonne pratique. À ce titre, je peux ajouter `"files.insertFinalNewline": true` dans les settings pour avoir ça automatiquement. Il y a déjà `"files.eol": "\n"` mais visiblement c'est pas la même chose.